crates/typst/src/image.rs

```diff
@@ -435,25 +497,64 @@ impl SvgFontLoader for WorldLoader<'_> {
             .clone()
     }
 
-    fn fallback_family(&self) -> Option<&str> {
-        self.fallback_family.as_deref()
+    fn fallback_families(&self) -> EcoVec<String> {
+        self.fallback_families.borrow().clone()
+    }
+
+    fn find_fallback(
+        &self,
+        text: &str,
+        like: Option<FontInfo>,
+        font: &usvg::Font,
```

Comment:
The conversion from `usvg::Font` to `FontVariant` could perhaps be extracted to make this argument cheaper to clone and hash (which happens due to tracking).